### DISCUSSION TITLE
Notes Icon

### DISCUSSION THREAD

**Discussion by irazetti** [twilio-labs/paste#2784] [Request: Icon] [answered]
Hi!
My colleague @pratapdd made a pull request for a notes icon. 
I catchup with @kelseycavitt and we discuss the [Notes Icon] and she re designed it to re adapt it to pastes icon. 
This is the **[figma file](https://www.figma.com/file/CYVeyXYnTMKlnQJBRdEwvW/Timeline?node-id=1343%3A112890)** and I also attach the new icon and also the link to the initial pull request (https://github.com/twilio-labs/paste/pull/2769)
Thanks
![Note](https://user-images.githubusercontent.com/104209586/199024740-ed7c8ee7-1407-4280-90ca-75aea2ea73dd.svg)

**Comment by loreina** [comment]
Hi @irazetti! Could you share a bit more info including:
1. A link or screenshot of the icon in Streamline
2. A brief explanation or mockup of what you'll need it for
3. Whether this icon is a logo icon or a UI icon

For reference, [icon contribution guidelines](https://paste.twilio.design/introduction/contributing/icons) on our docs site on how to get an icon into Paste.

Once we get more details on the icon addition, the team will discuss and get back to you. Thanks!

**Reply by irazetti** [reply]
Hi @loreina!

1. The icon at the beginning was taken originally from Streamline and then after reviewing it with @kelseycavitt she tweak it to better match the paste UI. 

2. We need that icon for the timeline component in AION and will be used for the notes section.

3. This icon is a UI icon. 

I hope this information helps. 

Thanks!

**Comment by loreina** [comment]
Hey @irazetti, you can add the icon to Paste using our [self-service guidelines](https://paste.twilio.design/introduction/contributing/icons/#setting-up-an-icon-in-figma). Once the branch is up, you can share the link here and designers on our team will review it. Thanks!

**Reply by pratapdd** [reply]
Hi @loreina we already have PR raised https://github.com/twilio-labs/paste/pull/2803/files

//cc @irazetti

**Comment by irazetti** [comment]
Ok @loreina we will then do a new PR with the new icon. Thanks!

**Comment by pratapdd** [comment]
Hi @loreina  @irazetti  This is new PR now  https://github.com/twilio-labs/paste/pull/2803

Thanks

**Comment by SiTaggart** [comment, marked as answer]
Hi Friends,

The notes icon is now out in NPM for downloading.

Thanks
